code/modules/mob/living/simple_animal/abnormality/waw/generalb.dm

```diff
@@ -22,7 +22,7 @@
 		/datum/ego_datum/weapon/loyalty,
 		/datum/ego_datum/armor/loyalty
 		)
-	loot = /obj/item/clothing/suit/armor/ego_gear/praetorian
+	loot = list(/obj/item/clothing/suit/armor/ego_gear/praetorian) // Don't think it was dropping before, this should make it do so
```

Comment:
Can you add in a 
`stat_attack = HARD_CRIT`?
I thought that it was the default but it's not.